Make sure that setuptools_scm actually installs the module and entrypoint.

--- a/setup.cfg	2021-01-28 10:10:54.291152917 -0500
+++ b/setup.cfg	2021-01-28 10:13:12.191007896 -0500
@@ -61,3 +61,20 @@
 tag_date = 0
 tag_svn_revision = 0

+[options]
+packages = 
+	gitchangelog
+package_dir =
+	=src
+setup_requires =
+	setuptools
+	setuptools-scm
+
+[options.entry_points]
+console_scripts = 
+	gitchangelog = gitchangelog.gitchangelog:main
+
+[options.package_data]
+gitchangelog =
+	gitchangelog.rc.*
+	templates/**/*
